documentation/modules/camelk/pages/camel-k-eventing.adoc

```diff
@@ -17,62 +17,44 @@ Navigate to the tutorial chapter's folder {camelk-repo}:
 cd $TUTORIAL_HOME/{camelk-repo}
 ----
 
-[#deploy-camel-k-source]
-== Deploy Knative Eventing CamelSource
+[#install-camel-k-plugin]
+== Install the kn kamelet source plugin
 
-The `CamelSource` allows you use a Camel K integration as part of the Knative Eventing architecture. Simply speaking, you can make the Camel K integration act as a https://knative.dev/docs/eventing/sources/[Knative Event Source] and  send the Camel exchanges (OUT) through a Knative Event Sink.
+The `Kamelet` allows you use a Camel K integration as part of the Knative Eventing architecture. Simply speaking, you can make the Camel K integration act as a https://knative.dev/docs/eventing/sources/[Knative Event Source] and  send the Camel exchanges (OUT) through a Knative Event Sink.
```

Comment:
"allows you to use" ?

Reply:
Will rephrase that paragraph. I just re-read it and I do not like it that much :D looks confusing